src/rust/sdk/example/src/main.rs

```diff
@@ -0,0 +1,353 @@
+        let best_ask_price = best_ask_level.price;
+        place_limit_order_user_entry(econia_address, &e_apt, &e_usdc, market_id, &econia_address, Side::Bid, size_lots_of_base, best_ask_price - 1, Restriction::NoRestriction, SelfMatchBehavior::CancelMaker)?
+    } else {
+        place_limit_order_user_entry(econia_address, &e_apt, &e_usdc, market_id, &econia_address, Side::Bid, size_lots_of_base, max_bid_price_ticks_of_quote, Restriction::NoRestriction, SelfMatchBehavior::CancelMaker)?
+    };
+
+    Ok((bid_entry, ask_entry))
+}
+
+pub async fn report_best_price_levels(view_client: EconiaViewClient<'_>, market_id: u64) -> EconiaResult<()> {
+
+    let (best_bid_level, best_ask_level) = get_best_levels(view_client, market_id).await?;
+
+    if best_bid_level.is_none() && best_ask_level.is_none() {
+        println!("There is no eAPT being bought or sold right now");
+        return Ok(());
+    }
+
+    println!("Best price levels:");
+
+    if let Some(best_bid_level) = best_bid_level {
+        let best_bid_volume = best_bid_level.size;
+        let best_bid_price = best_bid_level.price;
+        println!("  Highest BID/BUY @ {best_bid_price} ticks/lot, {best_bid_volume} lots");
+    } else {
+        println!("  No open bids");
+    }
+
+    if let Some(best_ask_level) = best_ask_level {
+        let best_ask_volume = best_ask_level.size;
+        let best_ask_price = best_ask_level.price;
+        println!("  Highest ASK/BUY @ {best_ask_price} ticks/lot, {best_ask_volume} lots");
+    } else {
+        println!("  No open asks");
+    }
+
+    Ok(())
+}
+
+#[tokio::main]
+async fn main() -> EconiaResult<()> {
+    let args = Args::parse();
+
+    let Init { e_apt, e_usdc, faucet_address, faucet_client, econia_address, mut econia_client } = init(&args).await;
+
+    print_title!("Create a market for eAPT/eUSDC");
+
+    let lot_size = 10u64.pow(8 - 3); // eAPT has 8 decimals, want 1/1000th granularity
+    let tick_size = 10u64.pow(6 - 3); // eAPT has 6 decimals, want 1/1000th granularity
+    let min_size = 1;
+
+    let entry = register_market_base_coin_from_coinstore(econia_address, &e_apt, &e_usdc, &APTOS_COIN_TYPE, lot_size, tick_size, min_size).unwrap();
+
+    econia_client.submit_tx(entry).await?;
+
+    let market_id = econia_client.view_client().get_market_id_base_coin(e_apt.clone().into(), e_usdc.clone().into(), lot_size, tick_size, min_size).await?;
+
+    let market_id = if let Some(market_id) = market_id {
+        println!("Market created with ID: {market_id}");
+        market_id
+    } else {
+        panic!("Could not create market."); // Should not happen
+    };
+
+    wait_for_enter!();
+
+
+    print_title!("Set up account A");
+
+    let (account_address_a, mut econia_client_a) = account(&faucet_client, &args.url, econia_address.clone()).await;
+
+    fund(&e_apt, 10u64.pow(18), &mut econia_client_a, faucet_address).await?;
+    println!("Minted EAPT to {account_address_a}");
```

Comment:
It looks like there are still `EAPT` and `EUSDC` in here

How about case-sensitive find and replace with `eAPT`/`eUSDC` other than `CONSTANT_CASE_VARIABLE_NAMES`